.circleci/config.yml.in

```diff
@@ -610,9 +662,13 @@ jobs:
 
 
 workflows:
-  build:
+  lint:
```

Comment:
Since the lint workflows don't require `torchtext` to be installed, I've extracted them out of the unittest workflows.

Reply:
Thank you for doing this. That was certainly on the radar :-)